presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1489,6 +1492,7 @@ else if (partitionUpdate.getUpdateMode() == NEW || partitionUpdate.getUpdateMode
         return Optional.of(new HiveWrittenPartitions(
                 partitionUpdates.stream()
                         .map(PartitionUpdate::getName)
+                        .map(name -> name.isEmpty() ? UNPARTITIONED_ID : name)
```

Comment:
@raghavsethi  This will be the OperatorInfo in OperatorStats. So it's our internal convention about how to representation about how to represent unpartitioned table. Using a dedicated string instead of empty string should help downstream consumer :)